### DISCUSSION TITLE
app-1.1.5 missing in Windows

### DISCUSSION THREAD

**Discussion by Elanmarkowitz** [Chia-Network/chia-blockchain#5739] [Support]
On Windows 10, I installed chia and can use the GUI. However, when I follow the directions to find the chia CLI executable, the app-1.1.5 directory is not there. Under the chia-blockchain dir there is only a directory called packages which contains chia-blockchain-1.1.5-full.nupkg . Would appreciate any help as I cannot figure out how to get my public farmer and pool key, and would like to be able to use the CLI in general.

**Comment by Elanmarkowitz** [comment]
Solved. Found at C:\ProgramData\[USER]\chia-blockchain\app-1.1.5\resources\app.asar.unpacked\daemon\chia

**Comment by BrandtH22** [comment]
@Elanmarkowitz , based on your previous message it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
